daprdocs/content/en/getting-started/quickstarts/actors-quickstart.md

```diff
@@ -30,10 +30,17 @@ As a quick overview of the .NET actors quickstart:
 For this example, you will need:
 
 - [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
-- [.NET SDK or .NET 6 SDK installed](https://dotnet.microsoft.com/download).
 <!-- IGNORE_LINKS -->
 - [Docker Desktop](https://www.docker.com/products/docker-desktop)
 <!-- END_IGNORE -->
+- [.NET 6](https://dotnet.microsoft.com/download), [.NET 8](https://dotnet.microsoft.com/download) or [.NET 9](https://dotnet.microsoft.com/download) installed
+
+{{% alert title="Note" color="primary" %}}
```

Comment:
`{{% alert ... %}}` shortcode typically doesn't render when put between `{{% codetab %}}` shortcode... could you build the site locally and see if it renders for you?

Reply:
Oops - you caught me. Didn't test this one out locally. I'll see what I can get working on this.

Reply:
@WhitWaldo so far, we've used `> **Note:**` for notes within `codetab` shortcode. It does a soft block quote.

Reply:
I haven't had the time to circle back on this, but I'll try to get to it before the holidays!